ovos_core/intent_services/__init__.py

```
@@ -284,13 +92,15 @@
         """
         default_lang = get_message_lang(message)
         valid_langs = get_valid_languages()
+        valid_langs = [standardize_lang_tag(l)
+                       for l in valid_langs]
```

Ruff

Ambiguous variable name: `l`

(E741)